koku/masu/processor/__init__.py

```diff
@@ -108,6 +108,18 @@ def is_validation_enabled(account):  # pragma: no cover
     return UNLEASH_CLIENT.is_enabled("cost-management.backend.enable_data_validation", context)
 
 
+def is_managed_ocp_cloud_processing_enabled(account):
+    account = convert_account(account)
+    context = {"schema": account}
+    return UNLEASH_CLIENT.is_enabled("cost-management.backend.feature-cost-5129-ocp-cloud-processing", context)
+
+
+def is_managed_ocp_cloud_summary_enabled(account):
```

Comment:
I presume this is just a place holder for now? Since the current PR only populates the new Trino table right. Just making sure I'm not blind and missing the summary changes.

Reply:
Yeah, this one is just a placeholder and the summary changes will be in a following PR.